internal/commands/inspect_image.go

```diff
@@ -101,20 +113,56 @@ func inspectImageOutput(
 	}
 	var buf bytes.Buffer
 	localMirrors := getLocalMirrors(info.Stack.RunImage.Image, cfg)
+	processes := displayProcesses(info.Processes)
 	tw := tabwriter.NewWriter(&buf, 0, 0, 8, ' ', 0)
 	defer tw.Flush()
 	if err := tpl.Execute(tw, &struct {
 		Info         *pack.ImageInfo
 		LocalMirrors []string
+		Processes    []process
 	}{
 		info,
 		localMirrors,
+		processes,
 	}); err != nil {
 		return "", err
 	}
 	return buf.String(), nil
 }
 
+func displayProcess(p lifecycle.Process, d bool) process {
```

Comment:
This function name makes it sound like it should be printing (or generating a string). I guess it depends on whether you read "display" as a verb or an adjective. Maybe something like `renderProcess`? Though you might be able to make the same argument with that name (it's the best I could come up with on the spot).

Also, I feel like primitive parameters should have more descriptive names (the `lifecycle.Process` param is fine, but the meaning of the `bool` is hard to infer).